README.md

```diff
@@ -14,6 +14,18 @@ gem 'capistrano', '~> 3.1'
 gem 'capistrano-rails', '~> 1.1'
 ```
 
+Run the following command to install the gems:
+
+```ruby
```

Comment:
Sorry to nitpick but could you remove `ruby` from the backticks? Since this is a shell command and not Ruby code.